pom.xml

```diff
@@ -112,7 +112,7 @@
 
     <!-- Java versions -->
     <javaVersion>1.7</javaVersion>
-    <targetJavaVersion>1.6</targetJavaVersion>
+    <targetJavaVersion>1.7</targetJavaVersion>
```

Comment:
This should not be in the PR. Unless support was dropped for Hadoop 1 and CDH4, we can't rev the targetJavaVersion.